### PR TITLE
Update .gitpod and Dockerfile parts

### PR DESCRIPTION
**Pull request by geropl**

### CODE REVIEW

**Comment by orucky** [comment]
One question: so is it true that if I change a custom Dockerfile, on the next open of a workspace the 'workspace image' will be rebuilt? What exactly means 'to change' it - just the timestamp or we compare the contents?

**Comment by geropl** [comment]
Thanks @orucky! Currently we compare commit hashes.

File: doc/40_Configuration.md [hunk 1]
```diff
@@ -13,8 +13,21 @@ Gitpod workspaces come with good defaults, but of course not every workspace loo
 
 ## `.gitpod` File
 
-A workspace gets configured through a `.gitpod` file written in YAML syntax. There are three ways
-you can provide this file:
+A workspace gets configured through a `.gitpod` file written in YAML syntax. It has the following structure:
+
+```yaml
+# The Docker image to run your workspace in
+image: gitpod/workspace-full:latest
```
[the remark this answers is not in the document]
Comment (geropl) [reply doc/40_Configuration.md]: Ok. I replaced it with a generic placeholder.

**Comment by orucky** [comment]
Hmm ... I noticed that the syntax coloring is wrong for that Docker image example ... not in the preview in Theia, but in the result served by the HTTP server. Is that automatic or do we control it?

**Comment by orucky** [comment]
OK, so it's apparently because '>' used in the placeholder got interpreted as the beginning of a block text value in YAML. Quotes around solved the problem. Making a PR for that would be a bit heavy weight, I pushed that to master ...

**Pull request merged**